athena/__init__.py

```
@@ -0,0 +1 @@
+from .version import VERSION
```

Ruff (0.8.2)

`.version.VERSION` imported but unused; consider removing, adding to `__all__`, or using a redundant alias

(F401)